docs/sql-guide/functions/function-datetimefromparts.md

```diff
@@ -0,0 +1,55 @@
+---
+title: DATETIMEFROMPARTS()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+
+# DATETIMEFROMPARTS() function
+
+The `DATETIMEFROMPARTS()` function takes integer date and time arguments and returns timestamp.
+
+## Syntax
+
+```
+DATETIMEFROMPARTS(int_year, int_month, int_date, int_hour, int_minute, int_seconds, int_milliseconds)
+```
+
+## Arguments
+
+| Argument         | Data type | Description                                                                                     | Required | Further information |
+|------------------|-----------|-------------------------------------------------------------------------------------------------|---|---|
```

Comment:
You don't have to put spaces and lots and lots of dashes in here because the system auto-arranges the content

I find it's simplest to go with something like this:

```
| Argument | Data type | Description | Required | Further information |
|---|---|---|---|---|
```
This also helps when troubleshooting table rendering errors

Reply:
My IDE is acting smart by adding all these dashes. I'll remove these.